jsonrpc/http.go

```diff
@@ -18,7 +16,7 @@ type Http struct {
 	log  utils.Logger
 }
 
-func NewHttp(port uint16, methods []Method, log utils.Logger) *Http {
+func New(port uint16, methods []Method, log utils.Logger) *Http {
```

Comment:
We will probably have RPC over Websocket in some point. So we need the Http postfix there.